### PR TITLE
feat: add SilentSignin and AuthCallback components

### PR DESCRIPTION
**Pull request by wenxuan70**
Referring to the `SilentSignin` function of casnode, put the universal components in the SDK.

https://github.com/casbin/casnode/blob/d526c9ae001057a3976c7a16a4a4e9a359d85ed4/web/src/App.js#L606-L628

### CODE REVIEW

File: src/AuthCallback.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
```
Comment (hsluoyz) [inline src/AuthCallback.tsx:15]: license header for all files

File: src/SilentSignin.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
```
Comment (hsluoyz) [inline src/SilentSignin.tsx:15]: Why not .ts?

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,3171 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@ampproject/remapping@^2.1.0":
+  version "2.2.0"
+  resolved "https://registry.npmmirror.com/@ampproject/remapping/-/remapping-2.2.0.tgz#56c133824780de3174aed5ab6834f3026790154d"
```
Comment (hsluoyz) [inline yarn.lock]: yarnpkg.com

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (hsluoyz) [inline tsconfig.json:1]: Add README.md

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: src/SilentSignin.tsx (hunk 2 above)
Comment (wenxuan70) [reply src/SilentSignin.tsx:15]: .ts will cause compilation to fail.

**Pull request merged**